app/helpers/toolbar_helper.rb

```diff
@@ -0,0 +1,28 @@
+module ToolbarHelper
+  def toolbar(title:, subtitle: '', &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.